src/conversions.jl

```diff
@@ -5,6 +5,33 @@ export rescale
 
 using ..LeapSeconds
 
+"""
+   taiutc(jd1, jd2)
+
+Transform a two-part Julia date from `TAI` to `UTC`.
+
+# Example
+
+```jldoctest
+julia> tai  = Epoch{TAI}(2.4578265e6, 0.30477440993249416)
+2017-03-14T07:18:20.325 TAI 
+julia> AstronomicalTime.Epochs.taiutc(tai.jd1, tai.jd2)
+(2.4578265e6, 0.30434616919175345)
+```
+"""
+function taiutc(jd1, jd2)
+    LS = LeapSeconds.leapseconds(2.4578245e6)
```

Comment:
* You need to get the leap seconds for this specific Julian date. Since this is a two-part Julian date and the leapseconds function expect a single number, you need to add the two components.
* Local variables are always lowercase.
* `leapseconds` is exported. You do not need to prefix it with module name.

Reply:
Yes Sir